cscs-checks/prgenv/cuda/cuda_samples.py

```diff
@@ -33,11 +37,24 @@ def __init__(self):
             self.nvidia_sm = '60'
             self.modules = ['cudatoolkit']
 
+        output_patterns = {
+            'deviceQuery': r'Result = PASS',
+            'concurrentKernels': r'Test passed',
+            'simpleCUBLAS': r'test passed',
+            'bandwidthTest': r'Result = PASS',
+            'conjugateGradientCudaGraphs':
+                r'Test Summary:  Error amount = 0.00000'
+        }
+
         self.sourcesdir = 'https://github.com/NVIDIA/cuda-samples.git'
         self.build_system = 'Make'
         self.build_system.options = [f'SMS="{self.nvidia_sm}"',
                                      f'CUDA_PATH=$CUDA_HOME']
-        self.prebuild_cmds = ['git checkout v11.0']
+        self.prebuild_cmds = [f'git checkout v11.0', f'cd Samples/{test_name}']
+        self.executable = f'Samples/{test_name}/{test_name}'
+        self.sanity_patterns = sn.assert_found(
+            output_patterns[test_name], self.stdout
```

Comment:
Need to change these too
```suggestion
        self.prebuild_cmds = [f'git checkout v11.0', f'cd Samples/{self.test_name}']
        self.executable = f'Samples/{self.test_name}/{self.test_name}'
        self.sanity_patterns = sn.assert_found(
            output_patterns[self.test_name], self.stdout
```